aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -112,6 +156,19 @@
   - sudo git lfs pull
   - echo "git checkout -qf ..."
   - GIT_LFS_SKIP_SMUDGE=1 sudo git checkout -qf $COMMIT
+  
+  - SUBMODULES=$(git submodule | awk '{ print $2 }')
```

Comment:
Can you add the simulateBeam.yaml environment backup? And/Or should this be a passed in as a parameter rather than loading it all?

Reply:
Not sure what do you mean regarding backup? I now think it would be better to name this local variable in lower case to make it clear that it is not environment variable and make it consistent with other local variables. Will change now.

Regarding parameters the idea is that a user would pass one or several configs and then this script can automatically detect which submodules it needs to load, so we don't need to specify the same path twice, for a config and for a submodule.

Reply:
I misread this. I thought it was reading the variables from an environment variable in lambda.